### PR TITLE
Refactor/response

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
1. What changes were made? 
I refactored the users_controller.rb file's responses.

2. Any issues you ran into? 
No, I just edited the response text and nothing else.

3. Did you run any new migrations? 
I did not run any new migrations.

4. Current test coverage: 

5. Other notes here: 
Please let me know if you have any questions or concerns with the response texts that I edited.

### CODE REVIEW

**Review by larroyo1 — COMMENTED** [review]
Looks good to me. I'll let Alex merge the pr in case he finds something that needs to be changed

File: app/facades/elections_facade.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  def self.elections(location)
+    service = ElectionService.get_elections(location)[:contests]
+
+    all_elections = service.map do |data|
+        if data[:type] != 'Referendum'
+            Election.new(data)
+        end
+    end
+    elections = all_elections.reject {|elec| elec == nil}
+    return elections
```
Comment (AlexMMcConnell) [inline app/facades/elections_facade.rb]: Returns are implicit - Don't need to specifically say 'return elections', can just say 'elections' instead

File: app/controllers/api/v1/users_controller.rb [hunk 2]
```diff
@@ -4,20 +4,20 @@ class Api::V1::UsersController < ApplicationController
...
+      render json: {error: "Error creating subscriber. A valid email must be provided."}, status: 400
     elsif user.save
       UserMailer.registration(user).deliver_now
-      render json: {success: "User created. A confirmation email has been sent to #{user.email}."}, status: 200
+      render json: {success: "You are now registered to receive notifications about upcoming elections in your state. A confirmation email has been sent to #{user.email}."}, status: 200
     else
-      render json: {error: "Error creating user."}, status: 400
+      render json: {error: "Error creating subscriber."}, status: 400
     end
   end
```
Comment (AlexMMcConnell) [inline app/controllers/api/v1/users_controller.rb:17]: Adjust tests for users controller to match new error/success messages

File: app/controllers/api/v1/users_controller.rb [hunk 3]
```diff
@@ -4,20 +4,20 @@ class Api::V1::UsersController < ApplicationController
...
 
   def destroy
     user = User.find(params[:id])
     user.destroy
 
-    render json: {success: "Successfully unsubscribed. User will no longer recieve emails at #{user.email}."}, status: 200
+    render json: {success: "You have successfully unsubscribed from Mi Voto, Mi Voz's email list. You will no longer receive email notifications at #{user.email}."}, status: 200
   end
```
Comment (AlexMMcConnell) [inline app/controllers/api/v1/users_controller.rb:24]: Similar case here - adjust tests to match new success message

Also create sad path if a user can't be found

File: app/facades/elections_facade.rb (hunk 1 above)
Comment (larroyo1) [reply app/facades/elections_facade.rb]: Nice, good call.

**Comment by larroyo1** [comment]
Everything looks good.

File: config/initializers/cors.rb [hunk 4]
```diff
@@ -5,12 +5,15 @@
...
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'http://localhost:3001'
```
Comment (elisebeall) [inline config/initializers/cors.rb:10]: @AlexMMcConnell @larroyo1  Hi 👋  Sorry to jump in with a review on code that has already been merged, but somehow this was switched again.  Front end is running on port 3000 please!

Comment (elisebeall) [reply config/initializers/cors.rb:10]: Screenshot from @ivonne-hernandez of what we need!
Screen Shot 2022-02-08 at 3 01 49 PM

File: app/controllers/api/v1/users_controller.rb [hunk 5]
```diff
@@ -3,21 +3,24 @@ class Api::V1::UsersController < ApplicationController
...
+    
     if !user.email.include?('@')
-      render json: {error: "Error creating user. A valid email must be provided."}, status: 400
+      render json: {error: "Error creating subscriber. A valid email must be provided."}, status: 400
     elsif user.save
       UserMailer.registration(user).deliver_now
-      render json: {success: "User created. A confirmation email has been sent to #{user.email}."}, status: 200
+      render json: {success: "You are now registered to receive notifications about upcoming elections in your state. A confirmation email has been sent to #{user.email}."}, status: 200
+    elsif User.find_by(email: user.email).present?
+      render json: {error: "This email is already susbscribed to receive election notifications."}, status: 400
```
Comment (elisebeall) [inline app/controllers/api/v1/users_controller.rb:13]: @larroyo1 @AlexMMcConnell could we please adjust this response to `{error: "This #{user.email} is already susbscribed to receive election notifications."}` ?

Our form will have already cleared all of the inputs, and maybe our user doesn't remember which email they had attempted to input.

Thanks!

**Pull request merged**